draft-lkspa-wimse-verifiable-geo-fence.md

```diff
@@ -215,7 +215,7 @@ Geo-location service (GL) runs outside of H -- besides the location from device
 * Agent is returned the attested composite location over the secure connection. Agent signs the attested composite location using TPM AK establishing proof of residency of composite location to H. This is called attested proof-of-residency aware composite location (APL).
 
 ## Step 2: Agent gets attested geographic boundary using Geo-fencing (GF) service
-* Geo-fence policies are of four flavours - (1)boolean membership of given boundary (rectangular, circular, state etc.), (2)precise location, (3)precise bounding box/circle of location, (4)approximate location (no definition of boundary). Thew first one, boolean membership of given boundary, is the most common and will be assumed as the default. They are available in the form of pre-defined templates or can be configured on demand. Enterprises, who are the user of the hosts, choose the geo-fence policies to be enforced for various hosts. Note that the hosts must belong to the set of trusted hosts in a shared ledger. The geo-fence policies applied to the set of trusted hosts are recorded in a shared ledger.
+* Geo-fence policies are of four flavours - (1)boolean membership of given boundary (rectangular, circular, state etc.), (2)precise location, (3)precise bounding box/circle of location, (4)approximate location with no definition of boundary. Thew first one, boolean membership of given boundary, is the most common and will be assumed as the default. They are available in the form of pre-defined templates or can be configured on demand. Enterprises, who are the user of the hosts, choose the geo-fence policies to be enforced for various hosts. Note that the hosts must belong to the set of trusted hosts in a shared ledger. The geo-fence policies applied to the set of trusted hosts are recorded in a shared ledger.
```

Comment:
```suggestion
* Geo-fence policies are of four flavours - (1) boolean membership of given boundary (rectangular, circular, state etc.), (2) precise location, (3) precise bounding box/circle of location, (4) approximate location with no definition of boundary.
The first one, boolean membership of given boundary, is the most common and will be assumed as the default. They are available in the form of pre-defined templates or can be configured on demand.
Enterprises, who are the user of the hosts, choose the geo-fence policies to be enforced for various hosts.
Note that the hosts must belong to the set of trusted hosts in a shared ledger.
The geo-fence policies applied to the set of trusted hosts are recorded in a shared ledger.
```

Reply:
Note (1), (2), etc... won't render as numbered bullets. use:
"1. my first bullet"
"1. my second bullet"
"1. my third bullet"
without the quote marks.